cli/Sources/TuistHasher/TargetScriptsContentHasher.swift

```diff
@@ -64,4 +84,63 @@ public struct TargetScriptsContentHasher: TargetScriptsContentHashing {
         }
         return try contentHasher.hash(stringsToHash)
     }
+
+    // MARK: - Private
+
+    private func processInputFileListPath(
```

Comment:
```suggestion
    private func hashInputFileListPath(
```

I'd consider returning an actual hash from here to improve the readability here.